### ISSUE TITLE
/rtp taking a long time

### ISSUE THREAD

**Issue by Dusten9752** [Zrips/CMI#7814]
### Description of Issue

After using the `/rtp` it's taking anywhere from 20-40 seconds to randomly teleport around the world.
[Video](https://youtu.be/6APNg06yIPQ)

### Version Information

```shell
CMI: 9.5.0.9 BungeeCord  SqLite
CMILib: 1.2.5.3 
Server: Paper(448) 1.19.3-R0.1-SNAPSHOT
CMI economy: Enabled Vault: 1.7.3-b131(CMIEInjector) CMI Chat: Disabled
```

### Errors

```shell
N/A
```

### Relevant Config Sections

```yaml
world:
      Enabled: true
      # Max coordinate to teleport, setting to 1000, player can be teleported between -1000 and 1000 blocks between defined center location
      # For example having centerX at 2000 and centerZ at 3000 while MaxRange is set to 1500 we will teleport player between x:500;z:1500 and x:3500;z:4500 coordinates
      MaxRange: 3000
      # If maxcord set to 1000 and mincord to 500, then player can be teleported between -1000 to -500 and 1000 to 500 coordinates
      MinRange: 100
      CenterX: 1000
      CenterZ: 5000
      Circle: false
      IgnoreWater: false
      IgnoreLava: false
      ignorePowderSnow: false
      # With this option we will only attempt to teleport player on highest block and ignore any cave teleportations
      surfaceOnly: true
      minY: -64
      maxY: 320
```

### Relevant Plugins

Vault 1.7.3

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by Otteee** [comment]
Have you already rendered your entire world with Chunky, for example? If that is not the case, he must also render into the place where you spawn.

**Comment by Dusten9752** [comment]
I have rendered all worlds with chunky completely.

**Comment by Dusten9752** [comment]
Fixed after 9.5.1.0
